yt/_typing.py

```diff
@@ -6,6 +6,12 @@
 FieldDescT = Tuple[str, Tuple[str, List[str], Optional[str]]]
 KnownFieldsT = Tuple[FieldDescT, ...]
 
+FieldType = str
+FieldName = str
+FieldKey = Tuple[FieldType, FieldName]
```

Comment:
grumble grumble I feel like we went back and forth on this exact thing a couple months ago and the changes might even still be in an outstanding PR ;)

Reply:
I know, sorry about that. I have come to like this style too since I watched https://www.youtube.com/watch?v=wbohVjhqg7c